frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/WorkerLifeCycle.java

```diff
@@ -45,6 +49,20 @@ public WorkerLifeCycle(ConfigManager configManager, Model model) {
         this.model = model;
     }
 
+    ArrayList<String> getEnvString(HashMap<String, String> envMap) {
+        ArrayList<String> envList = new ArrayList<>();
+        HashSet<String> blackList = configManager.getBlacklistEnvVars();
+
+        if (!blackList.contains("ALL")) {
```

Comment:
if contains all, returning empyt list?

Reply:
I agree. Probably I should remove all because ALL can potentially be a valid environment variable. Should I replace it with MMS_ALL... I will remove all for now.